CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/CloseAppPlugin.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
@mimosa2 I don't think we need this committed to the code base. I believe you used this for testing?

Reply:
Agree with @arcturus2, this can probably be omitted